.github/workflows/compose-maven-verify.yaml

```diff
@@ -137,13 +142,16 @@ jobs:
         # yamllint enable rule:line-length
       - name: Build code with Maven
         # yamllint disable rule:line-length
-        uses: Best-Quality-Engineering/ci-maven-action@v1
-        with:
-          phases: ${{ inputs.MVN_PHASES }}
-          pom-file: ${{ inputs.MVN_POM_FILE }}
-          profiles: ${{ inputs.MVN_PROFILES }}
-          maven-opts: ${{ inputs.MVN_OPTS }}
-          settings-file: settings.xml
+        run: |
+          echo "Maven build starting"
+
+          export ${{ inputs.MVN_OPTS }}
```

Comment:
Should this be `export MVN_OPTS=${{ inputs.MVN_OPTS }}` ?

Reply:
I wonder if we even need this, we later use the inputs command which the value of the MVN_OPTS variables is already available to be used from the inputs. I think we can remove this.